mlflow/utils/autologging_utils/versioning.py

```diff
@@ -1,7 +1,7 @@
 import importlib
 import yaml
 
-from distutils.version import LooseVersion
+from packaging.version import Version
```

Comment:
`packaging` is not a built-in package. We need to include it in MLflow's dependencies.

Reply:
ok. so shall we change to packaging.version ?